compute/oslogin/physical_security_key_ssh.py

```diff
@@ -55,15 +55,23 @@ def main(user_key, ip_address, dryrun, directory=None):
         .getLoginProfile(name="users/{}".format(user_key), view="SECURITY_KEY")
         .execute()
     )
-    security_keys = profile.get("securityKeys")
 
     if "posixAccounts" not in profile:
         print("You don't have a POSIX account configured.")
+        print("Did you enable OSLogin on your compute instance?")
         return
 
     username = profile.get("posixAccounts")[0].get("username")
 
     # Write the SSH private key files.
+    security_keys = profile.get("securityKeys")
+
+    if security_keys is None:
+        print("It seems that the account you are using to authenticate does not have any security keys assigned to it.")
+        print("Please check your Application Default Credentials (https://cloud.google.com/docs/authentication/application-default-credentials).")
+        print("More info about security keys: https://support.google.com/accounts/answer/6103523?visit_id=637673282586358398-2383089289&rd=1")
```

Comment:
IIUC, this support page describes only about physical security keys. Should we link it to this page instead which talks about the security keys and connecting to VMs with OS Login enabled? - https://cloud.google.com/compute/docs/oslogin/security-keys